config/obsoletion.yml

```
@@ -40,3 +40,9 @@ split:
     alternatives:
       - RSpec/SpecFilePathFormat
       - RSpec/SpecFilePathSuffix
+
+removed:
+  RSpec/Capybara/FeatureMethods:
+    reason: >
+      this cop has migrated to `RSpec/Dialect`. Please use `RSpec/Dialect` instead.
+      in .rubocop.yml file, enable and `RSpec/Dialect`.
```

Looking at [the style used in rubocop/rubocop](https://github.com/rubocop/rubocop/blob/08a7244081df110c8513584c156ff31950218757/config/obsoletion.yml#L64-L81), I think this should be edited just a little bit:
```suggestion
      this cop has migrated to `RSpec/Dialect`. Please use `RSpec/Dialect` instead
```